<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_rule">
     <attribute name="title">
      <string>filter rules</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_rule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="del_pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>80</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>del</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="accept_radioButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>330</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>accept</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="reject_radioButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>360</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>reject</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apply_pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>130</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>apply</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>300</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>default:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_nat">
     <attribute name="title">
      <string>nat settings</string>
     </attribute>
     <widget class="QPushButton" name="applynat_pushButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>350</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delnat_pushButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>300</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>del</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addnat_pushButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>240</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_nat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>60</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>140</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>firewall ip:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Intranet segment:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_statu">
     <attribute name="title">
      <string>statu links</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_flashlink">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>flash</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_log">
     <attribute name="title">
      <string>filter logs</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_flashlog">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>flash</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_logs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
